tests/InputFiles/ExportCoordinates/Input1D.yaml

```diff
@@ -6,13 +6,29 @@
 # ExpectedOutput:
 #   ExportCoordinates1DVolume0.h5
 
+Evolution:
+  InitialTime: 0.0
+  InitialTimeStep: 0.01
+  TimeStepper:
+    AdamsBashforthN:
+      Order: 1
+
 DomainCreator:
-  Interval:
-    LowerBound: [0]
-    UpperBound: [1]
-    IsPeriodicIn: [false]
-    InitialRefinement: [1]
-    InitialGridPoints: [3]
+    Interval:
+      LowerBound: [-0.5]
+      UpperBound: [0.5]
+      IsPeriodicIn: [false]
+      InitialRefinement: [2]
+      InitialGridPoints: [5]
```

Comment:
Why are you changing the default domain? just keep it as-is except for the `TimeDependence`. Also please don't change the indentation. (same for the other input files)